src/Knp/Component/Pager/Pagination/AbstractPagination.php

```diff
@@ -105,11 +111,17 @@ public function getPaginatorOption(string $name): mixed
         return $this->paginatorOptions[$name] ?? null;
     }
 
+    /**
```

Comment:
How is this improving static analysis?

Reply:
These **inheritdoc** does not really improve static analysis but refers to the type definition on the PaginationInterface.

Reply:
Could be removed if that is the preferred way.

Reply:
Done!